smoke-test/tests/cypress/cypress/e2e/settings/homePagePost.js

```diff
@@ -0,0 +1,65 @@
+const announcementTitle = 'Test Announcement Title'
+const linkTitle = 'Test Link Title'
+const description = 'Add Description to post announcement'
+const linkURL = 'https://www.example.com'
+const imagesURL = 'https://www.example.com/images/example-image.jpg'
+
+const homePageRedirection = () => {
+    cy.visit('/')
+    cy.waitTextPresent("Welcome back,")
+}
+
+const addAnnouncement = (title, description) => {
+    cy.get('[id="posts-create-post"]').click({ force: true });
+    cy.waitTextPresent('Create new Post')
+    cy.enterTextInTestId("create-post-title", title);
+    cy.get('[id="description"]').type(description)
+    cy.get('[data-testid="create-post-button"]').click({ force: true });
+    cy.reload()
+    homePageRedirection();
+    cy.waitTextPresent(announcementTitle);
+}
+
+const addLink = (linkTitle, linkURL, imageURL) => {
```

Comment:
Please don't keep same name as global variable. Can cause confusion on what is being used.

Reply:
okay will change it, thanks!